plugins/python-build/bin/python-build

```diff
@@ -1486,6 +1487,29 @@ use_xcode_sdk_zlib() {
   fi
 }
 
+use_tcltk() {
+   # check args passed, use that otherwise check for homebrew tcl-tk
+   local tcltk_libdir="$(brew --prefix tcl-tk 2>/dev/null || true)"
+   local regex_tcltk_libs_args="(--with-tcltk-libs='([^']+)')"
+
+   if [[ "$PYTHON_CONFIGURE_OPTS" =~ $regex_tcltk_libs_args ]]; then
+     local tcltk_libs_match="${BASH_REMATCH[1]}"
+     local tcltk_match_quoted="${tcltk_libs_match//--with-tcltk-libs=/}"
+     PYTHON_CONFIGURE_OPTS="${PYTHON_CONFIGURE_OPTS//$tcltk_libs_match/}"
+
+     echo "python-build: use tcl-tk from \$PYTHON_CONFIGURE_OPTS"
+     package_option python configure "--with-tcltk-libs='${tcltk_match_quoted}'"
+   elif [ -d "$tcltk_libdir" ]; then
+     # get the version from the folder that homebrew versions
+     local tcltk_version_long="$(ls "$(brew --cellar tcl-tk)")"
+     local tcltk_version="${tcltk_version_long%.*}"
+     local tcltk_flags="--with-tcltk-includes=-I$tcltk_libdir/include --with-tcltk-libs=-L$tcltk_libdir/lib -ltcl$tcltk_version -ltk$tcltk_version"
+     echo "python-build: use tcl-tk from homebrew"
```

Comment:
Using-from-Homebrew logic should probably be placed into a separate function like with other library searches.